mimesis/providers/person.py

```diff
@@ -176,49 +177,43 @@ def username(self, template: Optional[str] = None) -> str:
         :Example:
             Celloid1873
         """
-        # TODO: Optimize
-        _name = self.random.choice(USERNAMES).capitalize()
-
-        name = self.random.choice(USERNAMES)
         MIN_DATE = 1800
         MAX_DATE = 2070
-        date = self.random.randint(MIN_DATE, MAX_DATE)
 
-        templates = ('U-d', 'U.d', 'UU-d', 'UU.d', 'UU_d', 'U_d',
-                     'Ud', 'l-d', 'l.d', 'l_d', 'ld', 'default')
+        templates = ('U_d', 'U.d', 'U-d', 'UU-d', 'UU.d', 'UU_d',
+                     'ld', 'l-d', 'Ud', 'l.d', 'l_d', 'default')
+        DEFAULT_TEMPLATE = 'l.d'
```

Comment:
Move this above `templates`, please. Like this:

```
        MIN_DATE = 1800
        MAX_DATE = 2070
        DEFAULT_TEMPLATE = 'l.d'

        templates = ('U_d', 'U.d', 'U-d', 'UU-d', 'UU.d', 'UU_d',
                     'ld', 'l-d', 'Ud', 'l.d', 'l_d', 'default')
```

Reply:
Will do.